vspace/src/main/webapp/WEB-INF/tiles/skeleton_bt4.jsp

```diff
@@ -23,9 +23,11 @@
 
     <!-- Custom styles for this template -->
     <link href="<c:url value="/resources/bootstrap-4.1.2/dashboard.css" />" rel="stylesheet">
+    <link href="<c:url value="/resources/multiselect/css/multiselect.css" />" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
     <script src="<c:url value="/resources/jquery/jquery-3.3.1.min.js" />" ></script>
     <script src="<c:url value="/resources/jquery/jquery-ui.min.js" />" ></script>
+    <script src="<c:url value="/resources/multiselect/js/multiselect.min.js" />" ></script>
```

Comment:
these are only needed on some of the pages, right? Let's include them where they are needed and not everywhere.